.github/workflows/quarkus.yml

```diff
@@ -104,23 +89,6 @@ jobs:
         key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ runner.os }}-maven-
-    - uses: actions/cache@v4
-      with:
-        path: ~/.mx
-        key: ${{ runner.os }}-mx-${{ hashFiles('**/suite.py') }}
-        restore-keys: |
-          ${{ runner.os }}-mx-
-    - name: Fetch LabsJDK
-      run: |
-        mkdir jdk-dl
-        ${MX_PATH}/mx --java-home= fetch-jdk --jdk-id labsjdk-ce-latest --to jdk-dl --alias ${LABSJDK_HOME}
-    - name: Build graalvm native-image
-      run: |
-        export JAVA_HOME=${LABSJDK_HOME}
-        cd substratevm
-        ${MX_PATH}/mx --native=native-image,lib:jvmcicompiler --components="Native Image,LibGraal" build
-        mv $(${MX_PATH}/mx --native=native-image,lib:jvmcicompiler --components="Native Image,LibGraal" graalvm-home) ${GRAALVM_HOME}
```

Comment:
The shared action [no longer builds libgraal](https://github.com/oracle/graal/pull/8510/files#diff-c9554432e3ab1889e1df4c55d958fac7693956a0886b3e03ad3380a38bd6d756R53). It's actually not used when running a Native Image build.

Reply:
That's interesting. When did that change? Last time I checked libgraal was being used by GraalVM CE see https://github.com/graalvm/mandrel/issues/288#issuecomment-923973995

Reply:
Nothing changed, what I meant is that the compilation stage in Native Image uses jargraal anyway (it adds additional AOT-specific phases etc that would otherwise require recompilation of libgraal). Yes, libgraal would probably improve performance a bit when compiling jargraal, but building it probably takes more time in CI than it saves.

Reply:
OK, so if I get this right:

* It's still being used (if present) by the native-image generator (the java process which runs on GraalVM) but it's just used to run that process and is not being used to AOT compile the native image.

* Since GraalVM with libgraal is being tested extensively in other places there is no reason for testing it in this context as well.